client-runtime/src/main/java/com/microsoft/rest/v2/annotations/DELETE.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.v2.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.METHOD})            // The context in which annotation is applicable i.e. this annotation (GET) can be applied only to methods
+@Retention(RetentionPolicy.RUNTIME)      // Record this annotation in the class file and make it available during runtime.
+public @interface DELETE {
+    String value() default "";
```

Comment:
I think this is the same for @GET which is allowing @Url parameters to specify the entire path